AdminWebsite/AdminWebsite/ClientApp/src/app/edit-participant/edit-participant-search-results/edit-participant-search-results.component.ts

```diff
@@ -12,12 +12,18 @@ import { PageUrls } from 'src/app/shared/page-url.constants';
 export class EditParticipantSearchResultsComponent {
     @Input() contactEmail: string;
     @Input() result: ParticipantEditResultModel;
+    @Input() isUnauthorisedSearch: boolean;
     constructor(private service: ParticipantEditService, private router: Router) {}
 
     get userNotFound() {
         return this.result === null;
     }
 
+    get warningText(): string {
+        // tslint:disable-next-line: quotemark
+        return this.isUnauthorisedSearch ? 'Judge accounts cannot be edited' : "Sorry, we can't find a user with that name.";
```

Comment:
different quotation marks :P

Reply:
one of the left is the correct format, the right side is because the autoformatting doesn't like \'

it's inconsistent, I know